packages/database/src/migrations/1739970132204-ensureSystemUserPresent.ts

```diff
@@ -0,0 +1,20 @@
+import type { QueryInterface } from 'sequelize';
+
+export async function up(query: QueryInterface) {
+  // The previous iteration of that migration, 1685403132663-systemUser, was too
```

Comment:
Is that more correct
```suggestion
  // The previous iteration of this migration, 1685403132663-systemUser, was too
```